api/src/main/java/org/pmiops/workbench/google/CloudStorageServiceImpl.java

```diff
@@ -56,6 +56,13 @@ public String readMandrillApiKey() {
     return mandrillKeys.getString("api-key");
   }
 
+  @Override
+  public String getMoodleApiKey() {
+    JSONObject mandrillKeys = new JSONObject(readToString(getCredentialsBucketName(), "moodle-keys.json"));
```

Comment:
Is it important that this file is a JSON file? Or would just reading a plain-text string be simpler?

(I'm all for consistency, so if you say it's better for us to be consistent w/ how we store Mandrill keys, I'm fine with that! Just update the API keys section in our Moodle integration docs to describe the JSON structure, so the next person who has to regenerate keys knows what format they should be in.)

Reply:
I was debating this myself to make it like invitation-key or mandrill. I think i will revert it to be like invitation-key which is a text as we have currently just one element.

Reply:
We did start down the path of json-ifying configs, but I don't think it's necessary.